From 7c7250c6d4ae6c2433dc046083d2a6b98c5688bd Mon Sep 17 00:00:00 2001
From: Volodymyr Zhdanov <wight554@gmail.com>
Date: Sun, 17 Mar 2019 21:31:18 +0000
Subject: [PATCH 342/420] ASoC: msm8998: kill logspam if max wsa devices
 property is set to 0

* I'm not sure it's a good idea to print this messages dozens of times
* Also remove useless xiaomi's hack (it acts the same as property anyway)

Change-Id: Ib12e7c793aa53e757356edefec950906641d8f1a
---
 sound/soc/msm/msm8998.c | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/sound/soc/msm/msm8998.c b/sound/soc/msm/msm8998.c
index b348ba8ddb79..1da77be3fb64 100644
--- a/sound/soc/msm/msm8998.c
+++ b/sound/soc/msm/msm8998.c
@@ -9419,13 +9419,10 @@ static int msm_init_wsa_dev(struct platform_device *pdev,
 		dev_dbg(&pdev->dev,
 			 "%s: wsa-max-devs property missing in DT %s, ret = %d\n",
 			 __func__, pdev->dev.of_node->full_name, ret);
-#ifdef CONFIG_MACH_XIAOMI_MSM8998
-		ret = 0;
-#endif
 		goto err_dt;
 	}
 	if (wsa_max_devs == 0) {
-		dev_warn(&pdev->dev,
+		dev_dbg(&pdev->dev,
 			 "%s: Max WSA devices is 0 for this target?\n",
 			 __func__);
 		goto err_dt;
-- 
2.15.0

